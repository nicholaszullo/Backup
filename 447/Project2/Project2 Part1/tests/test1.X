v2.0 raw
00003039
000010e1
00000d5d
