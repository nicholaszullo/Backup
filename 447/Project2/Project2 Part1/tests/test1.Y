v2.0 raw
000849f8
00000097
00000035
