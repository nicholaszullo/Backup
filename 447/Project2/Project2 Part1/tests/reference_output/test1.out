0000 0000	0	1000 0111 1010 0011 0001	.... .... .... ....
0000 0001	0	0000 0000 1000 0001	.... .... .... ....
0000 0010	0	0000 1101 0111 1101	.... .... .... ....
